### PR TITLE
Hybrid functions: Promises support, polyfill Babel artifacts

### PR DESCRIPTION
**Pull request by inikulin**
\cc @AlexanderMoskovkin @helen-dikareva

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
The `[API] Hybrid function Should polyfill Babel artifacts:` test failed

**Comment by inikulin** [comment]
FPR. Unstable mobile client test failed.

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
\сс @AlexanderMoskovkin @helen-dikareva

File: src/client/driver/command-executors/execute-hybrid-fn-command.js [hunk 1]
```diff
@@ -4,19 +4,21 @@ import DriverStatus from '../status';
...
-
-            resolve(new DriverStatus({ isCommandResult: true, result }));
-        }
-        catch (err) {
-            // TODO proper error handling
-            resolve(new DriverStatus({ isCommandResult: true, result: err.message }));
-        }
-    });
+        })
+        .then(fn => {
```
Comment (AlexanderMoskovkin) [inline src/client/driver/command-executors/execute-hybrid-fn-command.js]: maybe should use concise syntax?
```js
.then(fn => fn.apply(window, command.args))
.then(result => ...)
```

**Comment by AlexanderMoskovkin** [comment]
just a style remark, lgtm in others

File: src/client/driver/command-executors/execute-hybrid-fn-command.js (hunk 1 above)
Comment (inikulin) [inline src/client/driver/command-executors/execute-hybrid-fn-command.js]: yes, good catch

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm
Should we call @VasilyStrelyaev to review?

**Comment by inikulin** [comment]
@helen-dikareva nope, there were no new comments introduced

**Pull request merged**